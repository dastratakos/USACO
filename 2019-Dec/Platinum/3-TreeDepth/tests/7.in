20 110 531627143
