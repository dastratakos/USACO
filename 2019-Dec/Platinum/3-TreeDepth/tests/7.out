521698138 54185119 383790995 125356097 186996122 507946400 519492165 6290825 371285862 81568004 351223842 523847702 287006750 172524202 316711790 370641576 417864003 15960335 298360142 55649023
